applications/serial_lte_modem/doc/CARRIER_AT_commands.rst

```diff
@@ -319,10 +429,17 @@ The ``<cmd>`` command is a string, and can be used as follows:
   ``<apn>`` must be a string in double quotes.
   For details, see the :kconfig:option:`CONFIG_LWM2M_CARRIER_CUSTOM_APN` Kconfig option.
 
+* ``AT#XCARRIERCFG="auto_register"[,<0|1>]``
+
+  Set a flag to automatically register to the device management server when attaching to the network.
+  When this configuration is disabled, the user must trigger the registration manually through the ``AT#XCARRIER="regup"`` command after recovering network coverage.
+  This command accepts two possible input parameters: ``0`` to disable or ``1`` to enable.
+
 * ``AT#XCARRIERCFG="auto_startup"[,<0|1>]``
 
   Set a flag to automatically apply the enabled settings to the LwM2M carrier library configuration and connect to the device management network.
   This command accepts two possible input parameters: ``0`` to disable or ``1`` to enable.
+  This command is not available when the :kconfig:option:`CONFIG_SLM_CARRIER_AUTO_STARTUP` Kconfig option is enabled.
 
 * ``AT#XCARRIERCFG="bootstrap_smartcard"[,<0|1>]``
 
```

Comment:
in `AT#XCARRIERCFG="carriers"` the default is no longer Any (or all) but all the carriers except generic.

Reply:
Will fix in a follow-up PR after the release.